concepticondata/conceptlists/SatterthwaitePhillips-2011-423.tsv

```diff
@@ -23,118 +23,118 @@ SatterthwaitePhillips-2011-423-21	665	BEE	21	171	bee	蜜蜂	29
 SatterthwaitePhillips-2011-423-22	2242	BEETLE	22	171	beetle	甲虫	386
 SatterthwaitePhillips-2011-423-23	2245	BELCH	23	171	belch	打嗝	278
 SatterthwaitePhillips-2011-423-24	2246	BELL	24	171	bell	铃	169
-SatterthwaitePhillips-2011-423-25	1251	BELLY	25	171	belly*	肚子	110
+SatterthwaitePhillips-2011-423-25	1251	BELLY	25	171	belly	肚子	110
 SatterthwaitePhillips-2011-423-26	1202	BIG	26	172	big	大	404
 SatterthwaitePhillips-2011-423-27	1759	OLDER BROTHER	27	172	big brother	哥哥	401
 SatterthwaitePhillips-2011-423-28	1758	OLDER SISTER	28	172	big sister	姐姐	407
-SatterthwaitePhillips-2011-423-29	937	BIRD	29	172	bird *	鸟	15
+SatterthwaitePhillips-2011-423-29	937	BIRD	29	172	bird	鸟	15
 SatterthwaitePhillips-2011-423-30	1403	BITE	30	172	bite	咬	325
 SatterthwaitePhillips-2011-423-31	887	BITTER	31	173	bitter	苦	266
 SatterthwaitePhillips-2011-423-32	163	BLACK	32	173	black	黑	65
-SatterthwaitePhillips-2011-423-33	946	BLOOD	33	173	blood*	血	127
-SatterthwaitePhillips-2011-423-34			34	173	blow (out)%	?	?
+SatterthwaitePhillips-2011-423-33	946	BLOOD	33	173	blood	血	127
+SatterthwaitePhillips-2011-423-34			34	173	blow (out)	?	?
 SatterthwaitePhillips-2011-423-35	176	BLOW (WITH MOUTH)	35	173	blow [of person]	吹	335
 SatterthwaitePhillips-2011-423-36	175	BLOW (OF WIND)	36	174	blow [of wind]	刮	351
 SatterthwaitePhillips-2011-423-37	837	BLUE	37	174	blue	蓝	71
 SatterthwaitePhillips-2011-423-38	1844	BOAT	38	174	boat	船	169,1
 SatterthwaitePhillips-2011-423-39	2493	BOIL	39	174	boil	煮（＝开水）	340
-SatterthwaitePhillips-2011-423-40	1394	BONE	40	174	bone*	骨头	126
+SatterthwaitePhillips-2011-423-40	1394	BONE	40	174	bone	骨头	126
 SatterthwaitePhillips-2011-423-41	994	BOW	41	175	bow	弓	165
 SatterthwaitePhillips-2011-423-42	1530	BOWL	42	175	bowl	碗	147
 SatterthwaitePhillips-2011-423-43	1366	BOY	43	175	boy	男孩 [B = 男青年]	253
-SatterthwaitePhillips-2011-423-44	1583	BRAIN	44	175	brain>>	脑子 [B= 脑髓]	119
+SatterthwaitePhillips-2011-423-44	1583	BRAIN	44	175	brain	脑子 [B= 脑髓]	119
 SatterthwaitePhillips-2011-423-45	1531	BRANCH	45	175	branch	树枝	53
-SatterthwaitePhillips-2011-423-46	680	BREAK (DESTROY OR GET DESTROYED)	46	176	break	弄坏 [B = pohuai]	317
-SatterthwaitePhillips-2011-423-47	2558	BREAK (CLEAVE)	47	176	break （branch）	断（小枝）	361
+SatterthwaitePhillips-2011-423-46	680	BREAK (DESTROY OR GET DESTROYED)	46	176	break (damage)	弄坏 [B = pohuai]	317
```

Comment:
I just realized that this is actually an instance of BREAK (SOMETHING) or even better DESTROY (SOMETHING). The Chinese word *could* be used intransitively, but usually, they'd add a passive marker to it, so this is really the action of destroying something. Following the English gloss BREAK (SOMETHING) should suffice.

Reply:
I'll note that, I was going to review the concept list later.